spec/compiler/formatter/formatter_spec.cr

```diff
@@ -1120,6 +966,41 @@ describe Crystal::Formatter do
       )
       end
       CRYSTAL
+
+    assert_format <<-CRYSTAL, flags: %w[def_trailing_comma]
```

Comment:
**question:** Why is this still using a flag?

Reply:
I wasn't really sure what to do with them, as the flags are now essentially no-ops. I can remove them if you'd rather but figured keeping them was less noise and made it more clear these specs related to that specific feature in the past.

There is also now some duplication in the specs since some previously existing non-flag specs will be testing the same thing, but I didn't think it was worth trying to deduplicate them.

Reply:
This notation sends the signal that this specific formatter behaviour depends on the `def_trailing_comm` flag.
But that's not true. So I think we should remove unused flags.